tests/fixtures/simple/swagger.yaml

```diff
@@ -45,6 +29,22 @@ paths:
           required: true
           type: string
           format: path
+  /greeting/{name}:
```

Comment:
TODO: move this back to its original position.

Reply:
The new order is actually needed since the routing behavior in the middleware was fixed and now correctly leverages Starlette routing. [Starlette route priority](https://www.starlette.io/routing/#route-priority) differs from Flask route priority:

> In cases where more that one route could match an incoming path, you should take care to ensure that more specific routes are listed before general cases.